webui/react/src/components/CheckpointModal.tsx

```diff
@@ -146,6 +147,19 @@ ${checkpoint?.totalBatches}? This action may complete or fail without further no
       { label: 'State', value: <Badge state={state} type={BadgeType.State} /> },
     ];
 
+    if (
+      config.integrations?.pachyderm !== undefined &&
+      glossaryContent.find(({ label }) => label === 'Data Input') === undefined
```

Comment:
when is this true? in the above array, there's only `Source` and `State` labels?